### PR TITLE
fine tuning example/starting point

### PR DESCRIPTION
**Pull request by hhuuggoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+> [!NOTE]
+> Sometimes the configuration delegates to other code/classes, for example
+> - load_datest, load_from_disk for referencing HuggingFace datasets
+> - UserAssistantFormat, VicunaFormat for PromptFormats
+> - ZeroShotQA and FewShotQA classes for Prompts
+>
+> These configurations are specified with a `method` and a `kwargs` value. `method` is a string that
+has been registered against an existing python function in code. You can also call methods that
+we haven't registered with the following syntax: `path.to.module::name`. The kwargs entry is a dictionary of
+> parameters that the method expects.
```
Comment (bhperry) [inline README.md:182]: Might be nice to also support `args` list. That way load_dataset and load_from_disk could both be used the same way.

**Review by bhperry — COMMENTED** [review]
Overall looks good

File: README.md (hunk 1 above)
Comment (hhuuggoo) [reply README.md:181]: yea i thnk that's a good idea - will save for the future

File: README.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+```
+Please summarize the following conversation
+User: Conversation: A: Hi Tom, are you busy tomorrow’s afternoon?
+B: I’m pretty sure I am. What’s up?
+A: Can you go with me to the animal shelter?.
+B: What do you want to do?
+A: I want to get a puppy for my son.
+
+Assistant: Summary: Tom and his friend are going to the animal shelter to get a puppy for Tom's son.
+```
```
[the remark this answers is not in the document]
Comment (bhperry) [reply README.md:164]: Should also talk about multi-round prompting (i.e. for few-shot and conversational prompts), and how strict formatting gets to be more important for longer conversations.

**Pull request merged**